openedx/core/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -38,7 +38,6 @@ def _assert_error(self, data, expected_error, expected_error_description):
             form.errors,
             {"error": expected_error, "error_description": expected_error_description}
         )
-        self.assertNotIn("partial_pipeline", self.request.session)
```

Comment:
Is this a regression - the partial pipeline is not deleted when an error occurs? Or why was it necessary to delete this line, and are we sure it's not testing something important?